exercises/acronym/runtests.jl

```diff
@@ -0,0 +1,42 @@
+# canonical test version 1.7.0
```

Comment:
```suggestion
# canonical data version: 1.7.0
```

The track maintenance dashboard parses this comment, so it needs to have a specific format.